frontend/src/container/CreateAlertChannels/index.tsx

```diff
@@ -108,7 +179,7 @@ function CreateAlertChannels({
 }
 
 interface CreateAlertChannelsProps {
-	preType?: ChannelType;
+	preType: ChannelType;
 }
 
 CreateAlertChannels.defaultProps = {
```

Comment:
please remove default props as `preType` is not optional props

Reply:
linter is treating preType as optional since it has default value "slack". 
Two ways to handle this: [1] Make preType as optional and set default value to slack [2] mark preType as required and pass parameter from the caller.

your response seems to suggest option [2]. Making the change. Please let me know if you meant something else.  

This is the change: 
// required 
preType: ChannelType 

In pages/alertchannelscreate:
	Component: (): JSX.Element => {
	return <CreateAlertChannels **preType="slack"** />;
}

* changes in bold

Reply:
Yes @mindhash

Reply:
Done. Fixed.